lib/utils/analytics/generatePayload.js

```diff
@@ -100,8 +100,13 @@ module.exports = async (serverless) => {
       plugins: serviceConfig.plugins ? serviceConfig.plugins.modules || serviceConfig.plugins : [],
       functions: Object.values(serviceConfig.functions).map((functionConfig) => {
         const functionEvents = functionConfig.events || [];
+        const functionRuntime = (() => {
+          if (functionConfig.image) return 'container_image';
```

Comment:
Maybe we can make more special name, e.g. `$containerimage` ? (I've added `$` to indicate it's not real runtime and removed space to maintain same convention with other runtimes, e.g. it's `nodejs` not `node_js`), WDYT

Reply:
I like that idea :+1: Changed